relay-config/src/config.rs

```diff
@@ -167,9 +168,20 @@ trait ConfigObject: DeserializeOwned + Serialize {
     /// The basename of the config file.
     fn name() -> &'static str;
 
-    /// The full filename of the config file, including the file extension.
+    /// Check the full filename of the config file if it exists in the file storage. If none of
+    /// the defined files exist, it returns the last one even if it does not exist
```

Comment:
By reading this comment it's not clear to me what the behavior is if a file exists (files are only checked?). What do you think about the following?
```suggestion
    /// The full filename of the config file, including the file extension. The first matching file is returned, falling back to the last extension if none match.
```